code/modules/mob/living/carbon/human/login.dm

```diff
@@ -1,5 +1,12 @@
 /mob/living/carbon/human/LateLogin()
 	..()
 	update_hud()
-	if(species) species.handle_login_special(src)
-	return
+	if(species)
+		species.handle_login_special(src)
+	if(mind.vampire && !(mind.vampire.status & VAMP_ISTHRALL))
+		mind.vampire.blood_hud = new /obj/screen/vampire/blood()
+		mind.vampire.frenzy_hud = new /obj/screen/vampire/frenzy()
+		mind.vampire.blood_suck_hud = new /obj/screen/vampire/suck()
+		client.screen += mind.vampire.blood_hud
+		client.screen += mind.vampire.frenzy_hud
+		client.screen += mind.vampire.blood_suck_hud
```

Comment:
I feel like this should be calling a proc under /datum/antagonist, so that different antagonist types can override it. 

Vampire specific stuff does not belong in `/mob/living/carbon/human` unless absolutely necessary.